reservacionVuelo/Screens/MyFlights.js

```diff
@@ -0,0 +1,75 @@
+import React, {Component} from 'react';
+import {View, Text, TouchableHighlight, ListView, FlatList} from 'react-native';
+import styles from '../components/styles';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import {FlightCard} from '../components/flightCard';
+
+const dummyData = [
```

Comment:
WDYT of having your dummy Data in a file maybe inside a folder to better identify it? I know that I really don't know how to use it and it's only for testing, but for the same reason I would recommend having it separate, so when you already have the DB implemented you just delete the file and you will not have to modify this file again.

Reply:
It was only temporaty, but now that we moved the data, the component look a lot cleaner!

Reply:
I'm happy to read that! 😊